### DISCUSSION TITLE
被ChatGPT暖暈了

### DISCUSSION THREAD

**Discussion by wu51000** [1101-GS4719-C/Python-Community-GS4719-C#1124] [科技新知]
文章中寫到「Chat GPT不關心你最終決定吃什麼，只關心你的健康」
太霸氣了吧……
會幫你決定要吃什麼之餘還顧及營養均衡
還作詞寫曲
你的另一半說不定都沒有chat gpt浪漫……
真是一個有趣又實用的聊天機器人
如果可以講笑話或給予情緒上的支持
或是新增一個不定時關心你生活日常的功能
這樣的話不僅可以給使用者驚喜感 還可以讓用戶有抒發心事的感覺
那真的會讓人暈沒了……
https://www.cheers.com.tw/article/article.action?id=5101461

**Comment by HsuChen88** [comment]
它會成為一個完美的伴侶，機器人超越人類。但如果是我問它問題去追女生就不一樣了，機器人去洗碗。

**Reply by wu51000** [reply]
🐸勒 你去買洗碗機比較快

**Comment by harrylee1971** [comment]
太暈了吧，平常沒人可以關心我但是AI會關心我，太贊了

**Reply by wu51000** [reply]
哎呀 同是天涯淪落人……

**Comment by ghost** [comment]
Chat GPT不關心你最終決定吃什麼，只關心你的健康，雖然單看這句話很暖心，但一想到他是經人為修正，使它的各項回答愈趨近於「人性化」，就感覺他只是在回答出最多人想聽的話，而不是在回答你的問題，感覺心又涼了一半。
